### ISSUE TITLE
Developer mode doesn't work with otel-launcher-go

### ISSUE THREAD

**Issue by arunk-s** [lightstep/otel-launcher-go#40]
**Describe the bug**
Lightstep developer mode as described on https://docs.lightstep.com/docs/use-developer-mode-to-quickly-see-traces doesn't work with `otel-launcher-go`

**To Reproduce**
Run the docker container of the Lightstep satellite as mentioned on this [link](https://docs.lightstep.com/docs/use-developer-mode-to-quickly-see-traces). 
Run like the following:
```go

		defaultTraceAttributes := make(map[string]string)
		defaultTraceAttributes[conventions.AttributeCloudRegion] = os.Getenv("REGION")
		defaultTraceAttributes[conventions.AttributeHostHostname] = *host

		launcherOptions := []otel_lanucher.Option{
			otel_lanucher.WithAccessToken("developer"),
			otel_lanucher.WithSpanExporterEndpoint("localhost:8360"),
			otel_lanucher.WithMetricExporterEndpoint("localhost:8360"),
			otel_lanucher.WithSpanExporterInsecure(true),
			otel_lanucher.WithMetricExporterInsecure(true),
			otel_lanucher.WithServiceName("my-server"),
			otel_lanucher.WithResourceAttributes(defaultTraceAttributes),
			otel_lanucher.WithLogLevel("debug"),
		}
		launcher := otel_lanucher.ConfigureOpentelemetry(launcherOptions...)
```

**Expected behavior**
The launcher should run sending the traces that should be visible on https://app.lightstep.com/my-company/developer-mode/

**Additional context**
First using the config as per the Lighstep docs won't work because of this line [here](https://github.com/lightstep/otel-launcher-go/blob/main/launcher/config.go#L235), which only accepts access token of certain lengths and none of them matches the length of string `developer`(9).
Next, I tried providing an empty password but that also doesn't work as I see the following error
 ```
2020/11/25 17:18:29 error: rpc error: code = Unimplemented desc = unknown service opentelemetry.proto.collector.trace.v1.TraceService
```

**Comment by codeboten** [comment]
hey @arunk-s, thanks for filing this issue. can you confirm what version of the developer satellite you have in your environment? Older versions of the satellite didn't support the OpenTelemetry protocol, which would have caused the error message you reported.

```
docker images | grep satellite
lightstep/developer-satellite                     latest                 f9ed5d7e0eed        2 months ago        326MB
```

**Comment by codeboten** [comment]
NOTE: developer mode doesn't currently support metrics yet, you can disable metrics by using `WithMetricsEnabled(false)`

**Comment by arunk-s** [comment]
Yes, it was an older version 🙈 
So I updated the satellite docker image and I can see in the debug log that the launcher started but I can't see any traces yet 😅 . 
I am following the docs [here](https://opentelemetry.lightstep.com/go/tracing/spans/) but they need update because `"go.opentelemetry.io/otel/api/global"` isn't available from version `0.14` onward.

So I've tried the following to create spans but I can't see them on the developer traces page.

```go
import (
	"go.opentelemetry.io/otel/baggage"
	"go.opentelemetry.io/otel/label"
	"go.opentelemetry.io/otel/trace"
        "io"
       "context"
)

var tracer trace.Tracer

func init() {
	tracer = otel.Tracer("my-server")
}

func main() {
	_, span := tracer.Start(context.Background(), "hello-span")
	time.Sleep(time.Second * 1)
	defer span.End()
}
``` 
I also tried setting up the otel `net/http` instrumentation as per the example [here](https://github.com/open-telemetry/opentelemetry-go-contrib/blob/master/instrumentation/net/http/otelhttp/example/server/server.go) but that also didn't work.

I'm using the following versions:
```
	go.opentelemetry.io/collector v0.7.0
	go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp v0.14.0
	go.opentelemetry.io/otel v0.14.0
	go.opentelemetry.io/otel/sdk v0.14.0
```

**Comment by arunk-s** [comment]
Also, is the accessToken going to be `""` for the developer mode ?

**Comment by marwan-at-work** [comment]
I opened a PR for this a while ago and this issue seems to still be open.  https://github.com/lightstep/otel-launcher-go/pull/60

Is this library still maintained at all? Thanks

**Comment by codeboten** [comment]
#60 was merged and released with 1.5.0, thanks for the PR @marwan-at-work!
